### PR TITLE
MTM-43469 Review Administration Users management

### PR DESCRIPTION
**Pull request by Airdroplet**
* fixed incorrect statements about search functionality in administration app
* fixed few places where links were not pointing to right place

### CODE REVIEW

File: content/users-guide/administration-bundle/managing-users.md [hunk 1]
```diff
@@ -44,11 +44,11 @@ A user list will be displayed, providing the following information for each user
 * The username that is used to access the tenant.
 * The name and email of the user, if set.
 * The global roles assigned to the user.
-* The [strength](/users-guide/getting-started/#login) of the password set for the user.
+* The [strength](/users-guide/getting-started/#to-change-your-password) of the password set for the user.
 
-To filter the list, you can use the search field at the left of the top menu bar. For details on the search functionality, refer to [Searching](/users-guide/getting-started/#searching) in the *Introduction*.
```
Comment (Airdroplet) [inline content/users-guide/administration-bundle/managing-users.md:49]: Please note for administration we don't have search feature. We only have filters for it. Therefor this line referencing to search documentation is not correct.

**Review by BeateRixen — DISMISSED** [review]

**Review by manasesjesus — DISMISSED** [review]

**Review by MWindel — DISMISSED** [review]

**Comment by BeateRixen** [comment]
@Airdroplet Should I merge this and should it go into the 10.13 docs or also earlier versions?

**Comment by Airdroplet** [comment]
@BeateRixen 10.13 only because I verified content only against that version and haven't checked older ones.  If from grammar perspective changes have been verified, please feel free to merge :)

File: content/users-guide/administration-bundle/managing-users.md (hunk 1 above)
Comment (BeateRixen) [reply content/users-guide/administration-bundle/managing-users.md:49]: @Airdroplet Actually, at the same place there is also a section on "Filtering". Added the link again and modified the target and wording. 
What is confusing to me is that the filter field shows a search icon. This is not really consistent.

**Review by BeateRixen — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Pull request merged**